index.js

```diff
@@ -15,9 +15,11 @@ module.exports = function (str, opts) {
 	var stats = [];
 	var separator = opts.separator || ' ';
 	var reSeparator = new RegExp(escapeStringRegexp(separator), 'g');
+	var headerLength = (lines[0] || '').length;
 
```

Comment:
Wouldn't it be better to do a pre-loop where we determine the longest line?

Then in the below we can normalize all lines to that length.